packages/plugin/src/theia.d.ts

```diff
@@ -6154,7 +6154,7 @@ declare module '@theia/plugin' {
      *
      * @sample `let sel:DocumentSelector = { scheme: 'file', language: 'typescript' }`;
      */
-    export type DocumentSelector = DocumentFilter | string | Array<DocumentFilter | string>;
+    export type DocumentSelector = DocumentFilter | string | ReadonlyArray<DocumentFilter | string>;
```

Comment:
Please revert these changes.

Reply:
Hi @msujew , thank you for kind remind. 
You are right, I added (https://github.com/eclipse-theia/theia/commit/ceaeee01bca2261a771e4066046b1ad4880ef8fb) by accident. the Line 6157 is the same now at master,  so don't need to reverse. It was changed at  in the previous issue #10070 .